websites/E/Exxen/presence.ts

```diff
@@ -0,0 +1,53 @@
+const presence = new Presence({
+    clientId: "794408060847390760"
+});
+
+const strings = presence.getStrings({
+    playing: "presence.playback.playing",
+    paused: "presence.playback.paused",
+    browsing: "presence.activity.browsing",
+    episode: "presence.media.info.episode"
+});
+
+/**
+ * Get Timestamps
+ * @param {Number} videoTime Current video time seconds
+ * @param {Number} videoDuration Video duration seconds
+ */
+function getTimestamps(
+    videoTime: number,
+    videoDuration: number
+): Array<number> {
+    var startTime = Date.now();
+    var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+    return [Math.floor(startTime / 1000), endTime];
+}
```

Comment:
It is merged to devHelper already. use ``presence.getTimestamps``

Reply:
Ok. I updated